README.md

```diff
@@ -1,5 +1,10 @@
 AutoFDO tools can be built on Ubuntu 20.04, 22.04 or CentOS 9, choose 1a or 1b to install prerequisites.
 
+> [!NOTE]
+> The Propeller codebase has moved to its own repository. For the most
```

Comment:
How about adding a date like "moved to its own repository (since Q1 2025)." or something like that so that when people read this they get an idea of how far out of date this repository may be?